docker/vnc/post-init.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
I believe you'll need to copy this into the image in the Dockerfile

Reply:
Gotcha